package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
+    "build": "electron-packager ./src matterfront --out=dist --ignore='^/dist$' --prune --asar --platform=all --arch=all --version=$npm_package_electronVersion"
```

Comment:
Sadly, this outputs `matterfront.app` on OS X, when we'd like to have `Matterfront.app`, but putting `Matterfront` there makes it capitalized for linux. Unless there's a trick in electron-packager to do some magic, I'd rather have it capitalized, as the linux guys will figure this out.